### DISCUSSION TITLE
How can I deal with 2 trade republic accounts (2 phonenumbers)

### DISCUSSION THREAD

**Discussion by Barriradio** [pytr-org/pytr#76] [Q&A]
It seams that pytr has problems to deal with 2 acounts. Somewere deep in the (windows) system pytr remember on one acount, even
if another login is used. pytr dl_docs gets the same data after the 2 different logins:

pytr login --phone +12345 --pin 1234
pytr dl_docs dir1
pytr login --phone +67890 --pin 5678
pytr dl_docs dir2

the data from dir1 and dir2 are mostly the same, coming from one account, why ?

**Comment by Katzmann1983** [comment]
This seems to be a bug (not switching properly). There is an open PR to fix this, but it still needs some fine tuning.
